vardict.pl

```diff
@@ -2390,8 +2400,20 @@ sub findOffset {
 # Adjust MNP when there're breakpoints within MNP
 sub adjMNP {
     my ($hash, $mnp, $cov, $chr, $REF, $sclip5, $sclip3) = @_;
-    while( my ($p, $v) = each %$mnp ) {
-	while( my ($vn, $mv) = each %$v ) {
+
```

Comment:
Added sort to process MNPs, other way counts for variants can be adjusted with different values between runs.